I gave nix-review another try but I think it'd take all day on my laptop.

While running it I did get the following errors though they don't seem related to this package so I'm a bit confused.

```
builder for '/nix/store/7sfljq22sax1spzlh1jk2kpkzy5s8ncf-breeze-qt5-5.15.5.drv' failed with exit code 2; last 10 log lines:
  Scanning dependencies of target breeze-settings5
  [ 10%] Building CXX object kstyle/config/CMakeFiles/breeze-settings5.dir/breeze-settings5_autogen/mocs_compilation.cpp.o
  [ 11%] Building CXX object kstyle/config/CMakeFiles/breeze-settings5.dir/main.cpp.o
  [ 12%] Linking CXX executable breeze-settings5
  /nix/store/mgdjnsrkqgmxqjaqaxgqyqm7fwyi96fk-binutils-2.31.1/bin/ld: /nix/store/s7fyj3h4z4h5r7mlwzhdvpk89kjmx785-kservice-5.58.0/lib/libKF5Service.so.5.58.0: undefined reference to `qt_version_tag@Qt_5.12'
  /nix/store/mgdjnsrkqgmxqjaqaxgqyqm7fwyi96fk-binutils-2.31.1/bin/ld: /nix/store/pll7y62wa1xzzvmhpjkd5jvjz2r2jqw2-kwidgetsaddons-5.58.0/lib/libKF5WidgetsAddons.so.5.58.0: undefined reference to `QCoreApplication::sendEvent(QObject*, QEvent*)@Qt_5'
  collect2: error: ld returned 1 exit status
  make[2]: *** [kstyle/config/CMakeFiles/breeze-settings5.dir/build.make:99: kstyle/config/breeze-settings5] Error 1
  make[1]: *** [CMakeFiles/Makefile2:645: kstyle/config/CMakeFiles/breeze-settings5.dir/all] Error 2
  make: *** [Makefile:130: all] Error 2
cannot build derivation '/nix/store/a7zm7zd5id30fk0spchqva8cxa0dgczn-kwin-5.15.5.drv': 1 dependencies couldn't be built
builder for '/nix/store/jx9z3sjln4aqb9n28ga6sq6n6qbl5625-python2.7-cheroot-6.5.6.drv' failed with exit code 1; last 10 log lines:
  0.98s teardown cheroot/test/test_ssl.py::test_tls_client_auth[0-True-localhost-pyopenssl]
  =========================== short test summary info ============================
  XFAIL cheroot/test/test_conn.py::test_Chunked_Encoding
    Server does not correctly read trailers/ending of the previous HTTP request, thus the second request fails as the server tries to parse b'Content-Type: application/json\r\n' as a Request-Line. This results in HTTP status code 400, instead of 413Ref: https://github.com/cherrypy/cheroot/issues/69
  XFAIL cheroot/test/test_conn.py::test_598
    Sometimes this test fails due to low timeout. Ref: https://github.com/cherrypy/cherrypy/issues/598
  XFAIL cheroot/test/test_core.py::test_parse_uri_unsafe_uri
    Fails on Python 2
  SKIPPED [1] cheroot/test/test_server.py:117: Abstract sockets don't work currently
  ==== 4 failed, 99 passed, 1 skipped, 3 xfailed, 1 warnings in 83.86 seconds ====
cannot build derivation '/nix/store/070040jmxzm7xby779bzs5xq9g8vcfd8-python2.7-cherrypy-17.4.2.drv': 1 dependencies couldn't be built
```
Further, is it really correct that this package has 4 different versions of `gst_plugins_good` and `evince` in its dependencies?

--- a/pkgs/applications/audio/strawberry/default.nix
+++ b/pkgs/applications/audio/strawberry/default.nix
@@ -1,0 +1,91 @@
+{ mkDerivation
+, stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkgconfig
+, alsaLib
+, boost
+, chromaprint
+, fftw
+, gnutls
+, libcdio
+, libmtp
+, libpthreadstubs
+, libtasn1
+, libXdmcp
+, pcre
+, protobuf
+, sqlite
+, taglib
+, libpulseaudio ? null
+, libselinux ? null
+, libsepol ? null
+, p11_kit ? null
+, utillinux ? null
+, qtbase
+, qtx11extras
+, qttools
+, withGstreamer ? true
+, gst_all_1 ? null
+, withVlc ? true
+, vlc ? null
+}:
+
+mkDerivation rec {
+  pname = "strawberry";
+  version = "0.6.3";
+
+  src = fetchFromGitHub {
+    owner = "jonaski";
+    repo = pname;
+    rev = version;
+    sha256 = "01j5jzzicy895kg9sjy46lbcm5kvf3642d3q5wwb2fyvyq1fbcv0";
+  };
+
+  buildInputs = [
+    alsaLib
+    boost
+    chromaprint
+    fftw
+    gnutls
+    libcdio
+    libmtp
+    libpthreadstubs
+    libtasn1
+    libXdmcp
+    pcre
+    protobuf
+    sqlite
+    taglib
+    qtbase
+    qtx11extras
+  ]
+  ++ lib.optionals stdenv.isLinux [
+    libpulseaudio
+    libselinux
+    libsepol
+    p11_kit
+    utillinux
+  ]
+  ++ lib.optionals withGstreamer (with gst_all_1; [
+    gstreamer
+    gst-plugins-base
+    gst-plugins-good
+  ])
+  ++ lib.optional withVlc vlc;
+
+  nativeBuildInputs = [ cmake pkgconfig qttools ];
+
+  cmakeFlags = [
+    "-DUSE_SYSTEM_TAGLIB=ON"
+  ];
+
+  meta = with lib; {
+    description = "Music player and music collection organizer";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ peterhoeg ];
+    # upstream says darwin should work but they lack maintainers as of 0.6.3
+    platforms = platforms.linux;
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -2783,6 +2783,8 @@ in
 
   sonobuoy = callPackage ../applications/networking/cluster/sonobuoy { };
 
+  strawberry = libsForQt5.callPackage ../applications/audio/strawberry { };
+
   tealdeer = callPackage ../tools/misc/tealdeer { };
 
   teamocil = callPackage ../tools/misc/teamocil { };

